docs/en/setup/backend/backend-storage.md

```diff
@@ -163,6 +166,37 @@ storage:
     advanced: ${SW_STORAGE_ES_ADVANCED:"{\"index.translog.durability\":\"request\",\"index.translog.sync_interval\":\"5s\"}"}
 ```
 
+### Specify Settings For Each Elasticsearch Index Individually
```

Comment:
So, we support other configurations too? Besides shards and replications. Because the original #9849 mentioned these two configurations only.